### PR TITLE
More details on error

### PR DESCRIPTION
**Pull request by achamayou**
#206

### CODE REVIEW

File: src/node/nodetonode.h [hunk 1]
```diff
@@ -75,7 +76,12 @@ namespace ccf
       auto& n2n_channel = channels->get(t.from_node);
 
       if (!n2n_channel.verify(hdr, asCb(t)))
-        throw std::logic_error("Invalid authenticated node2node message");
+      {
+        throw std::logic_error(fmt::format(
```
Comment (jumaffre) [inline src/node/nodetonode.h]: Could we add the same for `recv_encrypted()`?

Comment (achamayou) [reply src/node/nodetonode.h]: Certainly

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Pull request merged**